ChromeExtension/popup.html

```diff
@@ -1,11 +1,19 @@
 <!DOCTYPE html>  
 <html>
     <head>
-        <style>
-            button {height: 30px;width: 30px;outline: none;}      
-        </style>    
-    </head>;    
+        <link rel = "stylesheet" href="style.css">
+    </head>   
     <body>      
-        <button>About SustainaBrowse</button>    
+        <button id="aboutButton">About SustainaBrowse</button>
+        <button id="tips">View all past tips</button>
+        <button id="articles">View articles</button>
+        <button id="howTo">How to become more sustainable</button>
+        <script>
+            var button = document.getElementById("aboutButton");
+            button.addEventListener("click", function() {
+            // Your code goes here.
+            chrome.tabs.create({url: "https://www.google.ca/"});
+        });
+</script>

```

Comment:
nit: fix the indentation here

<script>
    var button = ...
    button.addEventListener("click", function() {
        chrome.tabs.create({...
    });
</script>

Reply:
Ok, thank you